src/entities/DistributedFactorGraph.jl

```diff
@@ -0,0 +1,22 @@
+import Base: show
+
+struct DistributedFactorGraph
+    # A cache - TBD
+    # something::T
+    # An API for communication
+    api::DFGAPI
+    # A series of mirrors that are asynchronously updated.
+    mirrors::Vector{DFGAPI}
```

Comment:
Does this relate to the mirroring happens inside or outside DFG conversation?  I’m nervous about building too much automation into DFG, vs DFG sets the API, establishes symmetry, provides set operations, and lays the ground work for “operating system” features.  Also need to understand how much serialization and deserialization needs to occur in DFG, my guess is at least some would have to occur here and the rest through dispatch to used defined converters.  If you imply mirroring/duplication, I’m just concerned about too much.  Goes back to discussion that duplication happens above DFG in either RoME or Caesar?

Reply:
This stuff is older code, saving it for moving it out later. Right now it's inert code though